CHANGELOG.md

```diff
@@ -664,7 +664,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ### Fixed
 
-- Actually create CASScrubRequestHeaders setting in _auth_cas.erb [#1730](https://github.com/puppetlabs/puppetlabs-apache/pull/1730) ([](https://github.com/))
+- Actually create CASScrubRequestHeaders setting in _auth_cas.epp [#1730](https://github.com/puppetlabs/puppetlabs-apache/pull/1730) ([](https://github.com/))
```

Comment:
This is an odd thing to change. I'd keep the changelog as it is, so it matches the actual PR title.

Reply:
sure